### ISSUE TITLE
Wrong font style display country specific

### ISSUE THREAD

**Issue by pkrolkgp** [humhub/humhub#5072]
### What steps will reproduce the problem?
Change language to PL
### What is the expected result?
Same font style display
### What do you get instead?
Weird looking country specific letters### Additional info

![obraz](https://user-images.githubusercontent.com/32438012/118096861-36deb080-b3d2-11eb-8a02-f2d3b73d35b2.png)

| Q                | A
| ---------------- | ---
| HumHub version   | 1.8.2 but works in every version

**Comment by luke-** [comment]
Probably there is an error in the translation. Can you please mark exactly which texts it relates to?

**Comment by pkrolkgp** [comment]
I dont think it relates to translation, because every "ą" "ę" "ł" seen in screenshot are from different font

**Comment by yurabakhtin** [comment]
@luke- @pkrolkgp I have tested this in different browsers:

### Windows10:

#### Chrome:
![windows10_chrome](https://user-images.githubusercontent.com/6995524/118118805-1aed0600-b3f6-11eb-8312-7322d200136b.png)

#### FireFox:
![windows10_firefox](https://user-images.githubusercontent.com/6995524/118118828-24766e00-b3f6-11eb-834d-5a0711363adf.png)

### MacOS:

#### Chrome:
<img width="1148" alt="macos_chrome" src="https://user-images.githubusercontent.com/6995524/118119067-78815280-b3f6-11eb-8a70-60de0dd20f89.png">

#### FireFox:
<img width="1155" alt="macos_firefox" src="https://user-images.githubusercontent.com/6995524/118119091-82a35100-b3f6-11eb-83dd-e26de1635625.png">

#### Safari:
<img width="1206" alt="macos_safari" src="https://user-images.githubusercontent.com/6995524/118119109-8b942280-b3f6-11eb-9c6f-6518bac580d5.png">

As we can see some browsers depending on OS have the issue.
I find we have `font-family: 'Open Sans',sans-serif;` for the text, if I remove the `Open Sans` from there, then all chars look similar, I guess the specific chars like `ą`, `ż`, `ł`, `ę` and etc. were not developed inside the font `Open Sans` but they exist in the next font `sans-serif` so they are loaded from the next font and it is why we see the difference.
I don't have an idea how to fix this, not sure that we can use `sans-serif` completely without `Open Sans` because the `sans-serif` looks bolder than `Open Sans`.

**Comment by smashm** [comment]
Maybe it has to do with different/new browser interpretation, how to handle diacritic support and the unicode-range.
More info: https://stackoverflow.com/questions/17922766/does-google-fonts-has-diacritic-diacritics
